eips_code/eip5450.py

```diff
@@ -1,16 +1,13 @@
 from dataclasses import dataclass
 from eip5450_table import TABLE, OP_RJUMP, OP_RJUMPI, OP_CALLF, OP_RETF
-from eip4750 import validate_code_section, immediate_sizes
+from eip4750 import validate_code_section, immediate_sizes, ValidationException
```

Comment:
Having multiple `ValidationException` definitions is a problem because they are different types. Let's always use `eip3540.ValidationException`.

Reply:
We'll have to remove these definitions from the code in EIPs text then, too. And import statement wouldn't be better there.

Reply:
https://github.com/ipsilon/eof/pull/19

Reply:
I think this is fine to remove these from EIP code.

Reply:
https://github.com/ethereum/EIPs/pull/5984